PullRequestCreated/PullRequestCreated.ts

```diff
@@ -1,21 +1,52 @@
 import { OctokitAction } from '../lib/OctokitAction';
 import { PullRequest } from '../lib/OctokitTypes';
-import { JIRA_ISSUE_PATTERN, JIRA_TASK_ISSUE } from '../lib/Constants';
+import { JIRA_ISSUE_PATTERN } from '../lib/Constants';
+
+interface IssueParameters {
+  type: string;
+  parent?: string;
+}
 
 class PullRequestCreated extends OctokitAction {
   protected async execute(): Promise<void> {
     const pr = await this.getPullRequest(this.payload.pull_request.number);
     if (this.shouldCreateIssue(pr)) {
+      const issueParameter = await this.newIssueParameters(pr);
       const projectKey = this.getInput('jira-project');
-      const issueKey = await this.jira.createIssue(projectKey, JIRA_TASK_ISSUE, pr.title);
-      await this.updatePullRequestTitle(this.payload.pull_request.number, `${issueKey} ${pr.title}`);
-      await this.updatePullRequestDescription(this.payload.pull_request.number, `${issueKey}\n\n${pr.body || ''}`);
+      const issueKey = await this.jira.createIssue(projectKey, issueParameter.type, pr.title, { parent: { key: issueParameter.parent } });
+      if (issueKey != null) {
+        await this.updatePullRequestTitle(this.payload.pull_request.number, `${issueKey} ${pr.title}`);
+        await this.updatePullRequestDescription(this.payload.pull_request.number, `${issueKey}\n\n${pr.body || ''}`);
+      } else {
+        console.log('Unable to update PR title and description');
```

Comment:
To follow the logging design, let's do this login inside `createIssue`. Like that, it will be done once for all and we won't need to bother every caller with his own logging.
```suggestion
```